packages/shared/src/events/eventing.ts

```diff
@@ -6,7 +6,11 @@ export type EventMetadata = {
 
 export type CommonEventDetails = {
   providerName: string;
+  /**
+   * @deprecated Use `domain` instead
+   */
   clientName?: string;
+  domain?: string;
```

Comment:
```suggestion
  readonly domain?: string;
```

All these props should be `readonly` I think (build/test passes fine when I do that), but we'll probably have to wait for 2.0 to fix that. Might as well fix this one now though I guess?